docs/cloud/index.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: default
+title: lakeFS Cloud
+description: This section includes lakeFS cloud documentation
+nav_order: 80
+has_children: true
+---
+
+# lakeFS Cloud
+
+{: .no_toc }
+
+[lakeFS Cloud](https://lakefs.cloud) is a managed cloud-based data lake versioning platform that helps organizations manage and analyze large amounts of data. It is designed to enable data engineers and data scientists to store, process, and analyze data in a single, scalable platform.
+
+lakeFS Cloud is a fully-managed lakeFS solution, implemented using our best practices, providing high availability, auto-scaling, support and enterprise-ready features.
+	
+## Features only available in lakeFS Cloud
+
+* [Role-Based Access Control](../reference/rbac.md)
+* [Auditing](./auditing.md)
+* [Single-Sign-On](./sso.md)
+    * SAML
+    * OIDC
+    * AD FS
+    * Okta
+    * Azure AD
+* [Managed Garbage Collection](./managed-gc.md)
+* [Private-Link](./private-link.md)
+* SOC 2 Type II Compliance
+
+## How lakeFS Cloud interacts with your infrastructure
```

Comment:
I like the diagram but it's still entirely unclear to me what's running where. This diagram is probably more detailed than it needs to be. As a user I'd like to see basically what compute runs on lakeFS' infra and what runs on mine, and what's connecting to my object store. 

You could use this existing diagram for a "lakeFS Architecture" reference page if you wanted. Or rework it to fit the above spec by using colours or bounding boxes to make it easier to grok.